code/utils/variance-per-voxel.py

```diff
@@ -0,0 +1,171 @@
+# If you want to show figures inside iPython notebook
+# %matplotlib inline
+
+# Importing modules and tools
+import numpy as np 
+import matplotlib.pyplot as plt 
+import nibabel as nib
+
+# import sys
+# Appending directoy to sys.path
+#sys.path.append('/Users/ying/Stat159/data/sub001/BOLD/task001_run001')
+# fname = bold_dico_dico7Tad2grpbold7Tad.nii
+
+def load_image(fname):
+	"""
+	Loads the .nii file 
+
+	Parameters
+	----------
+	fname : String
+	The path to the .nii file that we wish to load and get the data of
+
+	Returns
+	-------
+	data : a 4-D array
+	"""
+	img = nib.load(fname)
+	data = img.get_data() # data.shape yields (132, 175,  48, 451)
+	return data
+
+
+def isolate_vol(data, n):
+	"""
+	Selects the (n+1)th volume from the 4-D image data array by slicing over the
+	last dimension.
```

Comment:
Did you yapf format the file? I think this line is >80 chars. from main repo run: yapf -i code/utils/variance-per-voxel.py